src/index.ts

```diff
@@ -126,3 +126,71 @@ function generateStorybookFiles (options) {
     options
   })
 }
+
+function generateEjectFiles (configDir, options) {
+  const templatesRoot = path.resolve(__dirname, '../storybook')
+  this.addTemplate({
+    src: path.resolve(templatesRoot, 'eject', 'main.js'),
+    fileName: path.join(configDir, 'main.js'),
+    options
+  })
+  this.addTemplate({
+    src: path.resolve(templatesRoot, 'eject', 'preview.js'),
+    fileName: path.join(configDir, 'preview.js'),
+    options
+  })
+}
+
+export async function eject (options: StorybookOptions) {
+  const buildDir = path.resolve(options.rootDir, '.nuxt-storybook')
+  const configDir = path.resolve(options.rootDir, 'storybook')
+  if (!options.force && fs.existsSync(configDir)) {
+    logger.warn('Storybook is already ejected, use `--force` to overwrite files.')
+    return
+  }
+  const { loadNuxt, getBuilder } = requireMaybeEdge('nuxt')
```

Comment:
Do we need all of this complexity to initiate nuxt/builder to copy initial boilerplate similar to manual setup docs?

Reply:
Actually No😅
I'm going to refactor it. We just need to use `loadNuxt` and read `storybook` config.